pages/track-claims/index.md

```diff
@@ -1,6 +0,0 @@
----
-title: Track Claims
-entryname: claims-status
-layout: page-react.html
-description: Track the status of your VA claims and appeals.
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-4ae61ffe386c9a0e77b8aaea009edb69R5.